baker/archival/ArchivalFileUtils.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs-extra"
+import path from "path"
+import { hashBase36, hashBase36FromStream } from "../../serverUtils/hash.js"
+
+const hashFile = async (file: string) => {
+    const stream = await fs.createReadStream(file)
+    return await hashBase36FromStream(stream)
+}
+
+export const hashAndWriteFile = async (
+    filename: string,
+    content: string,
+    archiveDir: string
```

Comment:
TODO for myself: clean up the `archiveDir` param - we don't really need it here
(same goes for below)